network/network_windows.go

```diff
@@ -329,6 +404,13 @@ func (nm *networkManager) newNetworkImplHnsV2(nwInfo *NetworkInfo, extIf *extern
 	if err != nil {
 		// if network not found, create the HNS network.
 		if errors.As(err, &hcn.NetworkNotFoundError{}) {
+			// in dualStackOverlay mode, need to add net routes to windows node
+			if nwInfo.IPAMMode == string(util.DualStackOverlay) {
```

Comment:
didn't we decide to use IPV6Mode instead of adding a new mode?
We are using IPV6Mode for epInfo as well. So I would suggest using IPV6Mode in nwInfo to maintain that consistency